### PR TITLE
0x801C03F2-Delta update

### PR DESCRIPTION
**Pull request by NagaCSC**
Primary SMTP and Email address mismatch -0x801C03F2

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ If the error occurs again, check the error code against the following table to s
...
 | 0x801C03EE | Attestation failed.                                                | Sign out and then sign in again. |
 | 0x801C03EF | The AIK certificate is no longer valid.                            | Sign out and then sign in again. |
-| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync).
+| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync). Also if no sync conflict. Please verify the "Mail /Email address" in AAD and the Primary SMTP address in the proxy address is the same.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:71]: Suggestion:
```suggestion
| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue, refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync). Also, if no sync conflict exists, please verify that the "Mail/Email address" in AAD and the Primary SMTP address is the same in the proxy address.
```

**Review by illfated — APPROVED** [review]
Looks good to me, but we may just want to check with @JohanFreelancer9 if there is anything left to fine tune, grammar-wise.

File: windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ If the error occurs again, check the error code against the following table to s
...
 | 0x801C03EE | Attestation failed.                                                | Sign out and then sign in again. |
 | 0x801C03EF | The AIK certificate is no longer valid.                            | Sign out and then sign in again. |
-| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync).
+| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue, refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync). Also, if no sync conflict exists, please verify that the "Mail/Email address" in AAD and the Primary SMTP address is the same in the proxy address.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:71]:
```suggestion
| 0x801C03F2 | Windows Hello key registration failed.                             | ERROR\_BAD\_DIRECTORY\_REQUEST. Another object with the same value for property proxyAddresses already exists. To resolve the issue, refer to [Duplicate Attributes Prevent Dirsync](https://docs.microsoft.com/office365/troubleshoot/administration/duplicate-attributes-prevent-dirsync). Also, if no sync conflict exists, please verify that the "Mail/Email address" in AAD and the Primary SMTP address are the same in the proxy address.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md (hunk 2 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:71]: Good catch. I overlooked the plural form in my suggestion.

**Review by mapalko — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**